doc/bitcoin-release-notes/release-notes-0.10.0.md

```diff
@@ -612,7 +612,7 @@ Tests:
 - `7357893` Prioritize and display -testsafemode status in UI
 - `f321d6b` Add key generation/verification to ECC sanity check
 - `132ea9b` miner_tests: Disable checkpoints so they don't fail the subsidy-change test
-- `bc6cb41` QA RPC tests: Add tests block block proposals
+- `bc6cb41` QA RPC tests: Add tests block proposals
```

Comment:
As this file is a history / log of past commit messages it should not be modified.